docs/rules/no-skip-test.md

```diff
@@ -2,7 +2,9 @@
 
 Translations: [Français](https://github.com/avajs/ava-docs/blob/master/fr_FR/related/eslint-plugin-ava/docs/rules/no-skip-test.md)
 
-It's easy to make a test skipped with test.skip() and then forget about it. It's visible in the results, but still easily missed.
+It's easy to make a test skipped with `test.skip()` and then forget about it. It's visible in the results, but still easily missed.
+
+This rule is automatically fixable, it will remove the `skip` test modifier. 
```

Comment:
```suggestion
This rule is automatically fixable, it will remove the `.skip` test modifier. 
```
for consistency with `.only`?

Reply:
I think it should be `.` instead of `,`.

Reply:
Looking at the rest of the code, there're a lot of other instances without the `.`. (eg: "Unknown test modifier `foo`.", "in a non-`cb` test function." ...)

Should all these be prefixed by a `.` ?

Reply:
> Should all these be prefixed by a . ?

Yes

Reply:
Same question with assertions ("`t.foo()`" => "Unknown assertion method `foo`")

Reply:
> Unknown assertion method `.foo`.